media-video/ffmpeg/ffmpeg6-6.0.recipe

```diff
@@ -0,0 +1,346 @@
+		lib:libass$secondaryArchSuffix
+		lib:libdav1d$secondaryArchSuffix
+		lib:libfdk_aac$secondaryArchSuffix
+		lib:libfontconfig$secondaryArchSuffix
+		lib:libfreetype$secondaryArchSuffix
+		lib:libfribidi$secondaryArchSuffix
+		lib:libgme$secondaryArchSuffix
+		lib:libgmp$secondaryArchSuffix
+		lib:libopenjp2$secondaryArchSuffix
+		lib:libopenmpt$secondaryArchSuffix
+		lib:libsnappy$secondaryArchSuffix
+		lib:libsoxr$secondaryArchSuffix
+		lib:libvpx$secondaryArchSuffix
+		lib:libwebp$secondaryArchSuffix
+#		lib:libx264$secondaryArchSuffix
+#		lib:libx265$secondaryArchSuffix
+		lib:libxml2$secondaryArchSuffix
+		lib:liblzma$secondaryArchSuffix
+		"
+fi
+
+PROVIDES_avdevice="
+	ffmpeg6${secondaryArchSuffix}_avdevice = $portVersionCompat
+	lib:libavdevice$secondaryArchSuffix = $libavdeviceVersionCompat
+	"
+REQUIRES_avdevice="
+	haiku$secondaryArchSuffix
+	lib:libavcodec$secondaryArchSuffix == $libavcodecVersion
+	lib:libavfilter$secondaryArchSuffix == $libavfilterVersion
+	lib:libavformat$secondaryArchSuffix == $libavformatVersion
+	lib:libavutil$secondaryArchSuffix == $libavutilVersion
+	lib:libSDL2_2.0$secondaryArchSuffix
+	"
+
+PROVIDES_tools="
+	ffmpeg6${secondaryArchSuffix}_tools = $portVersion
+	cmd:ffmpeg$secondaryArchSuffix = $portVersionCompat
+	cmd:ffplay$secondaryArchSuffix = $portVersionCompat
+	cmd:ffprobe$secondaryArchSuffix = $portVersionCompat
+	"
+REQUIRES_tools="
+	haiku$secondaryArchSuffix
+	lib:libavcodec$secondaryArchSuffix == $libavcodecVersion
+	lib:libavdevice$secondaryArchSuffix == $libavdeviceVersion
+	lib:libavfilter$secondaryArchSuffix == $libavfilterVersion
+	lib:libavformat$secondaryArchSuffix == $libavformatVersion
+	lib:libavutil$secondaryArchSuffix == $libavutilVersion
+	lib:libSDL2_2.0$secondaryArchSuffix
+	"
+CONFLICTS_tools="
+	ffmpeg${secondaryArchSuffix}_tools
+	ffmpeg5${secondaryArchSuffix}_tools
+	"
+
+PROVIDES_devel="
+	ffmpeg6${secondaryArchSuffix}_devel = $portVersionCompat
+	devel:libavcodec$secondaryArchSuffix = $libavcodecVersionCompat
+	devel:libavdevice$secondaryArchSuffix = $libavdeviceVersionCompat
+	devel:libavfilter$secondaryArchSuffix = $libavfilterVersionCompat
+	devel:libavformat$secondaryArchSuffix = $libavformatVersionCompat
+	devel:libavutil$secondaryArchSuffix = $libavutilVersionCompat
+	devel:libswresample$secondaryArchSuffix = $libswresampleVersionCompat
+	devel:libswscale$secondaryArchSuffix = $libswscaleVersionCompat
+	"
+REQUIRES_devel="
+	ffmpeg6$secondaryArchSuffix == $portVersion base
+	ffmpeg6${secondaryArchSuffix}_avdevice == $portVersion
+	devel:libspeex$secondaryArchSuffix
+	"
+CONFLICTS_devel="
+	ffmpeg${secondaryArchSuffix}_devel
+	ffmpeg5${secondaryArchSuffix}_devel
+	"
+
+# -- HACK! --
+# ffmpeg ~0.10 has perpetually had & caused problems when compiled with GCC 2.
+# (Later versions do not compile with GCC 2 at all.) So instead, we now compile
+# FFmpeg for x86_gcc2 with a modern GCC.
+#
+# This works because GCC's C ABI has not changed between 2 and 7; only the C++
```

Comment:
The same as above here.

Reply:
Left overs from previous patchset, will adjust :)